### DISCUSSION TITLE
Vite: error: Duplicate injected file

### DISCUSSION THREAD

**Discussion by LukePetko** [iTwin/itwinjs-core#4499] [Q&A] [answered]
Hi,

I'm using itwin libraries with vite. I used an example vite configuration mentioned here: https://github.com/iTwin/itwinjs-core/discussions/4003#discussion-4267807, to inject the process, but I get a following error:

```error: Duplicate injected file "node_modules/.pnpm/@esbuild-plugins+node-globals-polyfill@0.1.1/node_modules/@esbuild-plugins/node-globals-polyfill/_virtual-process-polyfill_.js"```

Have anyone met with this issue? 

Thanks

**Comment by kckst8** [comment]
Can you share your configuration?

**Reply by arnab-git-hub** [reply]
@LukePetko Have you figured out a way to resolve the issue at your end? If not, can you please share your configuration details with us.

**Comment by LukePetko** [comment, marked as answer]
In `@itwin/browser-authorization` 0.6.0 there is no env variable in the library anymore so this issue is solved and no polyfills are needed
